src/xml_dump.h

```diff
@@ -104,19 +107,27 @@ char *cr_xml_dump_primary(cr_Package *package, GError **err);
  */
 char *cr_xml_dump_filelists(cr_Package *package, GError **err);
 
+/** Generate filelists_ext xml chunk from cr_Package.
+ * @param package       cr_Package
+ * @param err           **GError
+ * @return              xml chunk string or NULL on error
+ */
+char *cr_xml_dump_filelists_ext(cr_Package *package, GError **err);
+
 /** Generate other xml chunk from cr_Package.
  * @param package       cr_Package
  * @param err           **GError
  * @return              xml chunk string or NULL on error
  */
 char *cr_xml_dump_other(cr_Package *package, GError **err);
 
-/** Generate all three xml chunks (primary, filelists, other) from cr_Package.
+/** Generate all three xml chunks (primary, filelists[_ext], other) from cr_Package.
  * @param package       cr_Package
+ * @param filelists_ext gboolean
  * @param err           **GError
  * @return              cr_XmlStruct
  */
-struct cr_XmlStruct cr_xml_dump(cr_Package *package, GError **err);
+struct cr_XmlStruct cr_xml_dump(cr_Package *package, gboolean filelists_ext, GError **err);
```

Comment:
Please create a new function.

Reply:
done (also not done in the Python side for the same reason)